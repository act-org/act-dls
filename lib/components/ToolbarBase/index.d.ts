/**
 * @prettier
 */
import * as React from 'react';
import { ToolbarProps } from '@material-ui/core';
export declare type Props = ToolbarProps;
declare const ToolbarBase: React.FC<Props>;
export default ToolbarBase;
//# sourceMappingURL=index.d.ts.map