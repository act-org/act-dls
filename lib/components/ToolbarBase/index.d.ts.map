{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/ToolbarBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAAW,YAAY,EAAE,MAAM,mBAAmB,CAAC;AAE1D,oBAAY,KAAK,GAAG,YAAY,CAAC;AAEjC,QAAA,MAAM,WAAW,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAEmB,CAAC;AAErD,eAAe,WAAW,CAAC"}