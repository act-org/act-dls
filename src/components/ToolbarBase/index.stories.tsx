/**
 * @prettier
 */

import * as React from 'react';
import { boolean, select } from '@storybook/addon-knobs';

import ToolbarBase from '.';

export const Base = (): React.ReactElement<any> => (
  <ToolbarBase
    disableGutters={boolean('Disable Gutters?', false)}
    variant={select(
      'Variant',
      {
        Dense: 'dense',
        Regular: 'regular',
      },
      'regular',
    )}
  >
    Toolbar
  </ToolbarBase>
);

export default {
  component: ToolbarBase,
  parameters: {
    info: {
      text: 'This is the base variant of the Toolbar component.',
    },
  },
  title: 'Atoms/Toolbar',
};
