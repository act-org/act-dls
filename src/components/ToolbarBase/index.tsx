/**
 * @prettier
 */

import * as React from 'react';
import { Toolbar, ToolbarProps } from '@material-ui/core';

export type Props = ToolbarProps;

const ToolbarBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <Toolbar {...props} />;

export default ToolbarBase;
