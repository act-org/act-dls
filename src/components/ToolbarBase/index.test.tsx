/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import ToolbarBase from '.';

describe('ToolbarBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <ToolbarBase disableGutters variant="regular">
        Toolbar Base
      </ToolbarBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
