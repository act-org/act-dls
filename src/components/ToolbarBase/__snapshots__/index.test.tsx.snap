// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolbarBase matches the snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root MuiToolbar-root MuiToolbar-positionFixed MuiToolbar-colorPrimary mui-fixed MuiPaper-elevation4"
  >
    Toolbar Base
  </header>
</div>
`;
